loadtest.py

```diff
@@ -0,0 +1,163 @@
+import base64
+import hmac
+import os
+import random
+import threading
+import uuid
+
+from requests.exceptions import HTTPError
+
+from fxa import errors
+from fxa.core import Client
+from fxa.plugins.requests import FxABrowserIDAuth
+
+from ailoads.fmwk import scenario, requests
+
+lock = threading.RLock()
+
+# Read configuration from env
+SERVER_URL = os.getenv('SYNCTO_SERVER_URL',
+                       "https://syncto.stage.mozaws.net:443")
+FXA_URL = os.getenv("SYNCTO_FXA_URL",
+                    "https://api-accounts.stage.mozaws.net/v1")
+FXA_USER_SALT = os.getenv("SYNCTO_FXA_USER_SALT")
+
+# if env variable present convert it to bytes
+if FXA_USER_SALT:
+    FXA_USER_SALT = FXA_USER_SALT.encode('utf-8')
+# else build a temporary one
+else:
+    FXA_USER_SALT = base64.urlsafe_b64encode(os.urandom(36))
+
+# Constants
+FXA_ERROR_ACCOUNT_EXISTS = 101
+
+ACCOUNT_CREATED = False
+
+
+def picked(percent):
+    """Should we stay or should we go?"""
+    return random.randint(0, 100) < percent
+
+
+class FXAUser(object):
+    def __init__(self):
+        self.server = FXA_URL
+        self.password = hmac.new(FXA_USER_SALT, b"syncto").hexdigest()
+        self.email = "syncto-%s@restmail.net" % self.password
+        print("Using FxA: ", self.email, ' : ', self.password)
+        self.auth = self.get_auth()
+
+    def get_auth(self):
+        with lock:
+            if ACCOUNT_CREATED:
+                return ACCOUNT_CREATED
+            return self.create_account()
+
+    def create_account(self):
+        global ACCOUNT_CREATED
+
+        # ONLY WORKS WITH FxA STAGE
```

Comment:
You mean like that? https://github.com/mozilla-services/ailoads-syncto/pull/1/files#diff-bcaef3aac1601d0e1e76a50ceb06a970R64